README.md

```diff
@@ -95,6 +96,7 @@
 
 | Presentation Title | Author(s) | Year |
 | --- | --- | --- |
+| [Static Analysis for Zero-Knowledge Programming Languages](https://www.youtube.com/watch?v=ST59slRf76I) | Filipe Casal | 2022 |
```

Comment:
Should have a folder with abstract info about the talk

Reply:
Removed presentations for now to get these reports published.